test/predictions/repo_test.exs

```diff
@@ -58,6 +59,67 @@ defmodule Predictions.RepoTest do
       end
     end
 
+    test "filtes out predictions with no departure" do
+      five_minutes_in_future = DateTime.add(Timex.now(), 5, :minute)
+
+      {:ok, five_minutes_in_future_string} =
+        Timex.format(five_minutes_in_future, "{ISO:Extended:Z}")
+
```

Comment:
Since you don't need the :ok and an error would just cause the test to fail you can do:

```
departure_time =
  Timex.now()
  |> DateTime.add(5, :minute)
  |> Timex.format!("{ISO:Extended:Z}")
```